user-guide/Advanced_Modules/User Defined APIs/UserDefinableApiEndpoint.md

```diff
@@ -0,0 +1,175 @@
+---
+uid: UD_APIs_UserDefinableApiEndpoint
+---
+
+# DataMiner UserDefinableApiEndpoint
+
+> [!WARNING]
+> The current feature is in preview and is not fully released yet. This feature should not be used in any staging or production environment.
+
+UserDefinableApiEndpoint is an extension module that runs an ASP .NET Core 5 web API. It handles the incoming API triggers over HTTP or HTTPS and sends the requests to the DataMiner agent(s) in a round robin-way.
+
+## Install
+
+Even though this is an extension module, it follows the release cycle of DataMiner. Meaning that the module will be installed and updated by a DataMiner upgrade package. Note that uninstalling the extension module will result in the feature being unusable.
+
+If the installation would be corrupted or the extension module is unintentionally uninstalled, you can install the extension module using the msi installer located in `C:\Skyline DataMiner\ModuleInstallers\DataMiner UserDefinableApiEndpoint <VERSION>.msi`. Admin rights are required.
+
+When downgrading the UserDefinableApiEndpoint, don't just run the installer of the lower version, but first uninstall the installed version. There is a known issue in MSI installers that corrupts the installation when downgrading. First uninstalling and then installing the lower version prevents this issue from happening.
+
+> [!IMPORTANT]
+> Downgrading DataMiner will not automatically downgrade the UserDefinableApiEndpoint extension module. So downgrading from a recent version to an old version could result in incompatibility between DataMiner and the extension module.
+
+## Logging
+
+The logging of the extension module can be found at the following location:
+
+`%ProgramData%\Skyline Communications\DataMiner UserDefinableApiEndpoint\Logs`
+
+Only core functionality errors will be logged. User errors like an empty route will not be logged as the exception passed to the user should give enough info. ASP.Net logging is managed by the Microsoft loglevel in the appsettings.
```

Comment:
'ASP.Net' is spelled with a space and different casing in line 10 (... ASP .NET Core 5...), I'm not sure which one is correct.